Sprint-1/exercises/count.js

```diff
@@ -1,6 +1,8 @@
 let count = 0;
 
-count = count + 1;
+count = count++;
```

Comment:
Why changed the original code at line 3?
If you want to use `++` operator, you can just write `count++` without the assignment operator.

Reply:
I have updated